File: game/game-frontend/src/app/model/game.ts
import {Grid} from "./grid";

export class Game {
  grid : Grid ;
  player_name : String;
  with_suggestion : boolean;
  coups: number;

  constructor(player_name: String, with_suggestion:boolean, grid:Grid) {
    this.player_name=player_name;
    this.with_suggestion=with_suggestion;
    this.grid=grid;
    this.coups=0;
  }
}

File: game/game-backend/src/main/java/game/model/Game.java
package game.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;


@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Game {
    String player_name;
    boolean with_suggestion;
    int coups;
    Grid grid;
}
